lib/parser.js

```diff
@@ -68,10 +77,48 @@ module.exports = class Parser {
         dockerFileName = build.dockerfile
       }
     }
+    if (Parser.isGitHubUrl(buildPath)) {
+      return path.resolve('/', dockerFileName)
+    }
     return path.resolve('/', buildPath, dockerFileName)
   }
 
   /**
+   * Parser for `code`
+   *
+   * @param {Object}          opts
+   * @param {String|Object}   opts.build
+   * @param {Object}          opts.warnings
+   * @return {Object|Null}    code object with `repo` and `commitish` which can be null.
```

Comment:
http://stackoverflow.com/questions/23303549/what-are-commit-ish-and-tree-ish-in-git#answer-23303550